shared/java/org/jetbrains/jwm/Window.java

```diff
@@ -57,6 +57,12 @@ public UIRect getContentRectAbsolute() {
     @NotNull @Contract("-> this")
     public abstract Window setIcon(File icon);
     public abstract Window setMouseCursor(MouseCursor cursor);
+    
+    /**
+    *  Sets window opacity (0 - 255)
+     */
+    @NotNull @Contract("-> this")
+    public abstract Window setOpacity(int opacity);
```

Comment:
- I want to propose float 0.0-1.0 instead of int value, to match Electron and macOS APIs
- Add getOpactiy method to complement setOpacity

Reply:
Ok. I'll change it.